Tasks/Xcode/task.loc.json

```diff
@@ -11,7 +11,7 @@
   "author": "Microsoft Corporation",
   "version": {
     "Major": 4,
-    "Minor": 127,
+    "Minor": 129,
```

Comment:
I bumped the minor version because it's a behavior change to pass an empty plist to Xcode when we don't find an embeddedprofile. Could affect Xcode 7 builds and the error messages you get when you need to enable iOS signing.

Reply:
Looks good, this shouldn't need a new major version.